packages/xdl/src/tools/Plist/build.ts

```diff
@@ -0,0 +1,150 @@
+/* (The MIT License)
```

Comment:
Might be better to create a new package since XDL is already pretty big

Reply:
yeah let's move this out into a separate package, maybe call it `@expo/plist`